features/support/helpers.rb

```diff
@@ -8,6 +8,14 @@ def load_all_apps
   @renewals_app = RenewalsApp.new
 end
 
+def visit_registration_details_page(reg_identifier)
```

Comment:
I like this - think it will make the tests quicker!

Reply:
yes! And if we can find a way to get the value of the registration's `_id` from the suite, then we can use a similar strategy to get directly into pages like finance-details as well which can be useful as well.